articles/nat-gateway/troubleshoot-nat.md

```diff
@@ -14,16 +14,6 @@ ms.author: allensu
 
 This article provides guidance on how to correctly configure your NAT gateway and troubleshoot common configuration and deployment related issues.  
```

Comment:
Those are anchor links that take you to the different sections in the article.  We have those for ease of finding the information when a customer first hits the article.